linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/bash/shared.sh

```diff
@@ -1,8 +1,14 @@
 # platform = multi_platform_all
 . /usr/share/scap-security-guide/remediation_functions
 
-var_enable_krb5="yes"
-
+{{% if product in ["rhel8", "fedora"] %}}
+AUDISP_REMOTE_CONFIG="/etc/audit/audisp-remote.conf"
```

Comment:
@jan-cerny According to manual, the transport option is set in `/etc/audit/auditd.conf` file.

Reply:
Thank you. I had look into this again and I have found it's in both files.
Try `man auditd.conf` and `man audisp-remote.conf` and it's documented  in both of them.
Now I don't know. Should we check both files? Does one of them override the other?

Reply:
@yuumasato 

Based on discussion with Steve Grubb on the mailing list https://lists.fedorahosted.org/archives/list/scap-security-guide@lists.fedorahosted.org/thread/5PZF5UTWX3K3OGRH353SDNT5U5CJ4FB7/ , configuration in `/etc/audit/auditd.conf` would be for  the aggregating server rather than the client that is sending the data.

I think that this specific rule is about the client that off-loads the data to a server. Therefore `/etc/audit/audisp-remote.conf` should be a correct file in this rule.